pyproject.toml

```diff
@@ -3,7 +3,7 @@ requires = ["setuptools", "Cython", "numpy", "scipy"]
 build-backend = "setuptools.build_meta"
 
 # [project]
-# name = "regmodsm"
+# name = "spxmod"
 # version = "0.1.0"
 # description = "Regression model smoother"
```

Comment:
I think maybe "Coefficient-smooth regression models" might be better? We're not smoothing an existing regression model.

Reply:
Great! I change the description.